src/files/glob-source.js

```diff
@@ -18,7 +18,12 @@ const errCode = require('err-code')
  * @param {boolean} [options.preserveMtime] - preserve mtime
  * @param {number} [options.mode] - mode to use - if preserveMode is true this will be ignored
  * @param {import('ipfs-unixfs').MtimeLike} [options.mtime] - mtime to use - if preserveMtime is true this will be ignored
- * @yields {Object} File objects in the form `{ path: String, content: AsyncIterator<Buffer> }`
+ * @return {AsyncIterable<{
```

Comment:
this should be AsyncGenerator right ?

Reply:
yes, thx, `AsyncIterable` is not specific enough. changing

Reply:
done!